model/layer.js

```diff
@@ -3,24 +3,63 @@ const Validation = require('../utils/validation.js');
 
 class Layer extends ParchKey {
 
+    /**
+     * The Abstract components that can exist on this Layer.
+     *
+     * Represented as an array of Component objects.
+     *
+     * @since 1.0.0
+     *
+     * @type {Array}
+     */
+    components;
+
+    /**
+     * The Connections that exist on this Layer.
+     *
+     * Represented as a array of Connection objects.
+     *
+     * @since 1.0.0
+     *
+     * @type {Array}
+     */
+    connections;
+
+    /**
+     * Whether this Layer is valid.
+     *
+     * @since 1.0.0
+     *
+     * @type {boolean}
+     */
+    valid;
```

Comment:
Are we saving the valid for caching reasons or are we generating a new one each time we validate to return it? It seems like we have a value here but then we have a `let valid = ...` line in the validation function and this doesn't get set

Reply:
We don't save the valid state in the validation function. That was leftover from before. It's been removed.